critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
+  'use strict';
+
+  var clearClass = 'rating-clear',
```

Comment:
Since there's a closure, so the global environment won't be polluted and the code has been taken from a plugin, I think its wise to let it be the way author wrote it.